src/aat/resources/features/F-121/S-121.2.td.json

```diff
@@ -647,49 +647,56 @@
                   "display_context": "MANDATORY",
                   "label": null,
                   "hint_text": null,
-                  "show_condition": null
+                  "show_condition": null,
```

Comment:
plenty of duplication here. Can you use Abi's approach to remove duplication in test data please?

Reply:
Will raise a PR off develop branch to fix this.